From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: KaneGreen <737445366KG@Gmail.com>
Date: Mon, 17 Sep 2001 00:00:00 +0000
Subject: [PATCH] grub2 use O2

---
 package/boot/grub2/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/boot/grub2/Makefile b/package/boot/grub2/Makefile
index 46e3597..ec33b9a 100644
--- a/package/boot/grub2/Makefile
+++ b/package/boot/grub2/Makefile
@@ -84,7 +84,7 @@ HOST_MAKE_FLAGS += \
 	TARGET_RANLIB=$(TARGET_RANLIB) \
 	LIBLZMA=$(STAGING_DIR_HOST)/lib/liblzma.a
 
-TARGET_CFLAGS := $(filter-out -fno-plt,$(TARGET_CFLAGS))
+TARGET_CFLAGS := $(filter-out -fno-plt -O% -funroll-loops -ffunction-sections -fdata-sections -Wl%--gc-sections,$(TARGET_CFLAGS)) -O2
 
 define Host/Configure
 	$(SED) 's,(RANLIB),(TARGET_RANLIB),' $(HOST_BUILD_DIR)/grub-core/Makefile.in
